--- start-studio.sh.orig	2013-08-12 09:12:43.716310251 +0400
+++ start-studio.sh	2013-08-15 10:45:28.955178895 +0400
@@ -1,12 +1,9 @@
 #!/bin/sh
 
 # Path to binary
-DIR="$( cd "$( dirname "$0" )" && pwd )"
-
-# Make sure binary is runnable
-chmod 754 "$DIR"/studiow
+DIR="_BASEDIR_"
 
 # Run with libs
-export LD_LIBRARY_PATH="$DIR":"$LD_LIBRARY_PATH"
-"$DIR"/studiow "$@"
+export LD_LIBRARY_PATH="$DIR":"$DIR/_LIBDIR_":"$LD_LIBRARY_PATH"
+"$DIR"/bin/studiow "$@"
 
