--- start-studio.sh.orig	2013-08-12 09:12:43.716310251 +0400
+++ start-studio.sh	2013-08-12 09:15:33.746309524 +0400
@@ -1,12 +1,12 @@
 #!/bin/sh
 
 # Path to binary
-DIR="$( cd "$( dirname "$0" )" && pwd )"
+DIR="$( cd "$( dirname "$0" )/.." && pwd )"
 
 # Make sure binary is runnable
-chmod 754 "$DIR"/studiow
+#chmod 754 "$DIR"/studiow
 
 # Run with libs
-export LD_LIBRARY_PATH="$DIR":"$LD_LIBRARY_PATH"
-"$DIR"/studiow "$@"
+export LD_LIBRARY_PATH="$DIR":"$DIR/lib32":"$LD_LIBRARY_PATH"
+"$DIR"/bin/studiow "$@"
 
