--- start-eADesigner.sh.orig	2013-08-06 14:45:14.838440148 +0400
+++ start-eADesigner.sh	2013-08-06 14:47:40.528439526 +0400
@@ -1,12 +1,12 @@
 #!/bin/sh
 
 # Path to binary
-DIR="$( cd "$( dirname "$0" )" && pwd )"
+DIR="$( cd "$( dirname "$0" )/.." && pwd )"
 
 # Make sure binary is runnable
-chmod 754 "$DIR"/eADesigner
+#chmod 754 "$DIR"/eADesigner
 
 # Run with libs
-export LD_LIBRARY_PATH="$DIR":"$LD_LIBRARY_PATH"
-"$DIR"/eADesigner "$@"
+export LD_LIBRARY_PATH="$DIR/lib":"$DIR/lib32":"$LD_LIBRARY_PATH"
+"$DIR"/bin/eADesigner "$@"
 
